src/Truck.java

```diff
@@ -1,6 +1,16 @@
-// Класс для грузовиков
-public class Truck extends Car {
-    public void checkTrailer() {
+public class Truck extends Transport implements Vehicle {
+    public Truck(String modelName, int wheelsCount) {
+        super(modelName, wheelsCount);
+    }
+
+    @Override
+    public void service() {
+        System.out.println("Обслуживаем грузовик " + getModelName());
+        int wheelsCount = getWheelsCount();
+        for (int i = 0; i < wheelsCount; i++) {
+            System.out.println("Меняем покрышку " + (i+1));
```

Comment:
```suggestion
            System.out.println("Меняем покрышку грузовика " + (i+1));
```
чтобы подчеркнуть, что это не так, как для машины или велосипеда, даже если код один и тот же